tests/drivers/pwm/gpio_loopback/testcase.yaml

```diff
@@ -0,0 +1,27 @@
+common:
+  tags:
+    - pwm
+    - drivers
+  depends_on: pwm
+  filter: dt_compat_enabled("test-pwm-to-gpio-loopback")
+  harness: ztest
+  harness_config:
+    fixture: pwm_loopback
```

Comment:
This is new name for fixture, thus not existing in CI (at test-server configuration), thus those tests are now not executed on HW.
If new physical connection are not needed, then you only need to add fixture name do setups, very similar like this:
https://projecttools.nordicsemi.no/bitbucket/projects/NCS-TEST/repos/test-servers-configs/commits/68a142c1c1e64c4835e93cb2e7172dd3103f2c1c

Reply:
And then rerun to check if all results are present.

Reply:
I changed the name of the fixture to gpio_loopback